src/Components/Recommendations/bookRec.css

```diff
@@ -0,0 +1,48 @@
+.library-display {
```

Comment:
This css is also unnecessary once we use the reusable components created in other sprints!